### ISSUE TITLE
Return array size from XP array datarefs

### ISSUE THREAD

**Issue by JT8D-17** [X-Friese/FlyWithLua#69]
I'm trying to return the array size of a dataref by using the method documented in the SDK:

> Read a part of a single precision floating point array dataref. If you pass NULL for outVaules, the routine will return the size of the array, ignoring inOffset and inMax.

So in theory
> XPLMGetDatavf("sim/aircraft/overflow/acf_tank_X", NULL,0,2)
should return the size of the array (10 for the example dataref)

But instead, I get "Wrong arguments to function XPLMGetDatavf" as a console output.

The only accepted format of a XPLMGetDatavf call is 
> XPLMGetDatavf(inDataRef, inOffset, inMax)
with everything else producing the aforementioned "wrong arguments" error.

My use case is trying to get the size of an array dataref in XP so that I can iterate through it.

It would be nice to have this functionality implemented.

**Comment by zodiac1214** [comment]
@JT8D-17 did you ever find a solution? I am facing the same issue and because dataref is yet provided, I have no way to know the length

**Comment by JT8D-17** [comment]
No. I worked around this by writing my own accessor in Lua's foreign function interface to directly interface with the SDK library.
